packages/flutter_tools/lib/src/project.dart

```diff
@@ -456,6 +453,25 @@ class IosProject extends FlutterProjectPlatform implements XcodeBasedProject {
     return null;
   }
 
+  /// The bundle name of the host app, `My App.app`.
+  Future<String> get hostAppBundleName async {
```

Comment:
nit: avoid getters that do work: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#getters-feel-faster-than-methods

The way we usually set this up is to have a private `_ hostAppBundleName`, and then a `_compute HostAppBundleName` that initializes the field:

```
int _foo;
int _computeFoo();
int get foo => _foo ??= _computeFoo();
``

Reply:
Will swap `productBundleIdentifier` too.